packages/engineer/src/feature-dependency-graph.ts

```diff
@@ -0,0 +1,21 @@
+import type { Feature } from '@wixc3/engine-core';
+
+export const buildFeatureLinks = (entry: Feature, visitedFeatures: { [propName: string]: number }, level: number) => {
+    const res: Array<{
+        source: string;
+        target: string;
+    }> = [];
+    const deps = entry.dependencies as Array<Feature>;
+    for (const dep of deps) {
+        res.push({ source: entry.id, target: dep.id });
+        if (!(dep.id in visitedFeatures)) {
+            visitedFeatures[dep.id] = level + 1;
```

Comment:
I don't really understand what does this used for?
keeping for each dependency the "distance" from the root?

Reply:
Yes, it is used to build the clusters in the visualization

Reply:
so maybe name it `dependencyDistancesFromRoot`?